ironfish/src/rpc/routes/wallet/getNotes.ts

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { BufferUtils } from '../../../utils'
+import { ApiNamespace, router } from '../router'
+import { RpcWalletNote, RpcWalletNoteSchema } from './types'
+import { getAccount, serializeRpcWalletNote } from './utils'
+
+const DEFAULT_PAGE_SIZE = 100
+
+export type GetNotesRequest = {
+  account?: string
+  pageSize?: number
+  pageToken?: string
+}
+
+export type GetNotesResponse = {
+  notes: Array<RpcWalletNote>
+  nextPageToken: string | null
+}
+
+export const GetNotesRequestSchema: yup.ObjectSchema<GetNotesRequest> = yup
+  .object({
+    account: yup.string().trim(),
+    pageSize: yup.number().min(1),
+    pageToken: yup.string(),
```

Comment:
The name pageToken seems not very clear, should we call it a cursor or something like database object key/id

Reply:
Since this endpoint will be paginated each response will be a 'page'. To me it made sense to use page terms for pageSize, pageToken, and nextPageToken

Reply:
I'd probably lean toward calling these `pageCursor` and `nextPageCursor` instead -- it doesn't make a huge deal, but a few APIs I glanced at tend to use the term `cursor`:
https://developer.squareup.com/docs/build-basics/common-api-patterns/pagination
https://dev.twitch.tv/docs/api/guide/#pagination
https://shopify.dev/docs/api/usage/pagination-rest